packages/tailwindcss/src/compat/apply-config-to-theme.ts

```diff
@@ -135,6 +135,7 @@ export function keyPathToCssProperty(path: string[]) {
   if (path[0] === 'animation') path[0] = 'animate'
   if (path[0] === 'transitionTimingFunction') path[0] = 'ease'
   if (path[0] === 'fontFamily') path[0] = 'font'
+  if (path[0] === 'maxWidth') path[0] = 'container'
```

Comment:
Should this be just `width`? `--max-width` is still a namespace that is valid when used with the `max-width` helper but I think the intention is that the `width` namespace migrates to `container` by default?

Reply:
Oh I see in v3 the `max-w-*` `w-*` and `min-*` utilities don't share a dimension :D never mind me